scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  specialty CHARACTER VARYING
+  );
+
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  insurance CHARACTER VARYING
+  );
+
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  date TEXT,
+  time TEXT,
```

Comment:
You should probably steer clear of naming your columns `date` or `time`, since those are considered data types in SQL.